qe_metrics/commands/init_db/__init__.py

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
Please do not put code in `__init__.py` file

Reply:
^^https://github.com/RedHatQE/qe-metrics/pull/4#discussion_r1466434589 

Will move